.github/workflows/linter.yml

```diff
@@ -47,3 +47,10 @@ jobs:
       - name: Run Linter
         continue-on-error: true
         run: golangci-lint run ./...
+
+  python-lint:
```

Comment:
python-formatter? Do we plan to add a linter too?

Reply:
probably be good to add it as a precommit hook as well

Reply:
Guess it should be formatter instead of linter